### DISCUSSION TITLE
Problem only with wifi

### DISCUSSION THREAD

**Discussion by andicurgu** [SkyrilHD/Dell-E7250-Hackintosh#17] [Q&A]
I have problem with wifi. All other kext are ok. Can any help me how to configure wifi?

**Comment by SkyrilHD** [comment]
I suppose you talk about Intel WiFi.

Download the kext from here: [itlwm](https://github.com/OpenIntelWireless/itlwm) and paste it to the Kexts folder. Then add the kext "details" in the config too. You could use ProperTree which does it do automatically. Here is the guide on how to do it: [config setup](https://dortania.github.io/OpenCore-Install-Guide/config.plist/#adding-your-ssdts-kexts-and-firmware-drivers)

**Reply by andicurgu** [reply]
Thanks for help.
